tests/console/system_prepare.pm

```diff
@@ -131,6 +132,21 @@ sub run {
         my $regcode = (get_var('AGAMA_PRODUCT_ID') =~ /SAP/) ? get_var('SCC_REGCODE_SLES4SAP') : get_var('SCC_REGCODE');
         my $regurl = get_var('SCC_URL');
         assert_script_run("suseconnect -r $regcode --url $regurl");
+
+        # Regster HA extension and output some debug info for reference
+        if (get_var('SCC_ADDONS', '') eq 'ha') {
+            record_info('Regster HA extension');
+            assert_script_run('SUSEConnect' . ' -p '
```

Comment:
Why not using existing util method to register addons?
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/suseconnect_register.pm#L62C5-L62C21

Reply:
Sure, let me try `command_register()`.

Reply:
Updated code to use command_register: https://openqa.suse.de/tests/16552983#step/system_prepare/41 (passed)